lib/vtt.js

```diff
@@ -599,7 +607,7 @@ function CueStyleBox(window, cue, styleOptions) {
                                      : cue.vertical === "lr" ? "vertical-lr"
                                                              : "vertical-rl",
     unicodeBidi: "plaintext",
-    textAlign: cue.align === "middle" ? "center" : cue.align,
+    textAlign: cue.align === "center" ? "center" : cue.align,
```

Comment:
Isn't this the same as just `textAlign: cue.align,`?

Reply:
yeah, oops, this should stay `middle`